scripts/check_ipfs_hashes_pushed.py

```diff
@@ -31,9 +32,10 @@
 
 def check_ipfs_hash_pushed(ipfs_hash: str) -> bool:
     """Check that the given ipfs hash exists in the registry"""
+
     try:
         url = f"{IPFS_ENDPOINT}/{ipfs_hash.strip()}"
-        res = requests.get(url, timeout=120)
+        res = requests.get(url, timeout=10)
```

Comment:
not needed can revert back if desirable, was because some hashes were failing at one point and it was too slow for me to trouble shoot, didn't realize I forgot to revert :grimacing:

Reply:
Revert timeout please